mlir/lib/Target/LLVMIR/Dialect/OpenMP/OpenMPToLLVMIRTranslation.cpp

```diff
@@ -299,6 +299,22 @@ convertOmpWsLoop(Operation &opInst, llvm::IRBuilderBase &builder,
       break;
     }
 
+    if (loop.schedule_modifiers().hasValue()) {
```

Comment:
Do we need a TODO here also for multiple modifiers? Also in print and parse?

Reply:
The TODO was mainly to explain why there was commented out code. SIMD will just add a bit more code here... :)